boilerplate/openshift/golang-osd-operator/standard.mk

```
@@ -110,7 +110,14 @@ envtest: isclean
 	@eval $$($(MAKE) env --no-print-directory) || (echo 'Unable to evaulate output of `make env`.  This breaks osd-operators-registry.' >&2 && exit 1)
 
 .PHONY: test
-test: envtest gotest yaml-validate
+test: envtest gotest yaml-validate check-generate
+
+.PHONY: check-generate
+check-generate: 
+	@$(MAKE) -s isclean --no-print-directory || (echo 'check-generate requires a clean project to run. Please commit or revert you changes first.' >&2 && exit 1) 
```

```suggestion
	@$(MAKE) -s isclean --no-print-directory || (echo 'check-generate requires a clean project to run. Please commit or revert your changes first.' >&2 && exit 1) 
```
...but doesn't the `isclean` target already print an appropriate message? I guess this one is (slightly) more specific to `check-generate`, but still seems kind of redundant.

Another thing: if I've got `ALLOW_DIRTY_CHECKOUT` set in my env, this will pass, and then the `generate` below will potentially end me up in an unrevertable state. I wonder if we should force `ALLOW_DIRTY_CHECKOUT=false` in this rule. Thoughts?

---

I thought the `-s` would silence the output, but only it silences only the targets names in fact. Will remove it indeed as not worth the redundancy.